src/components/picker/picker.tsx

```diff
@@ -154,7 +154,7 @@ export class Picker {
             return;
         }
 
-        this.chipSet.setFocus(true);
+        this.chipSet?.setFocus(true);
```

Comment:
Whaa… I assume this incorporates an automatic null-check, but I've never seen it before. New js-magic? Or something in typescript or Stencil?

Reply:
It's new in Typescript 3.7, but it's also going to land in ES sometime in the future, I think the draft is in the final stages :slightly_smiling_face: